.github/workflows/pr-request.yaml

```diff
@@ -155,13 +154,4 @@ jobs:
       
       - name: Run addlicense
         id: license_header_check
-        run: addlicense  -c "Google LLC" -l apache -v -check .
-      
-      - name: run if license_header_check failed
-        if: failure() && steps.license_header_check.outcome == 'failure'
-        env:
-          PULL_REQUEST: ${{ github.event.number }}
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          echo "License header check failed.\nConsider running \`make add-license-headers\`.\n(Please check ./scripts/make/codegen.mk for more details.)" > /tmp/comment_body
-          gh pr comment "${PULL_REQUEST}" -F /tmp/comment_body
\ No newline at end of file
+        run: addlicense  -c "Google LLC" -l apache -v -check .
```

Comment:
nit: I'm a big fan of "text file should be ending with a new line"...

Reply:
Added a new line break.